modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -161,10 +161,9 @@ class SeqexecEngine(settings: SeqexecEngine.Settings) {
 
   private def loadEvents(seqId: SPObservationID): SeqAction[List[Event]] = {
     val t: EitherT[Task, SeqexecFailure, (List[SeqexecFailure], Option[Sequence[Action \/ Result]])] = for {
-      odbSeq       <- EitherT(Task.delay(odbProxy.read(seqId)))
-      _            <- EitherT(Task.delay(odbSeq.unExecutedSteps.fold(().right[SeqexecFailure], EmptySequence(odbSeq.title).left)))
-      progIdString <- EitherT(Task.delay(odbSeq.config.extract(OCS_KEY / InstConstants.PROGRAMID_PROP).as[String].leftMap(ConfigUtilOps.explainExtractError)))
-      progId       <- EitherT.fromTryCatchNonFatal(Task.now(SPProgramID.toProgramID(progIdString))).leftMap(e => SeqexecFailure.SeqexecException(e): SeqexecFailure)
+      odbSeq <- EitherT(Task.delay(odbProxy.read(seqId)))
+      _      <- EitherT(Task.delay(odbSeq.config.extract(OCS_KEY / InstConstants.PROGRAMID_PROP).as[String].leftMap(ConfigUtilOps.explainExtractError)))
```

Comment:
These lines are just checking that the program has an id though they are not being used. Are they necessary?